src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -226,6 +227,103 @@ public CloudioEndpoint(String uuidOrAppName) throws InvalidUuidException, Invali
         internal = new InternalEndpoint(uuidOrAppName, null, null);
     }
 
+    //TODO WIP Here
+    public void addNodesFromJson(String jsonPath) throws CloudioFactoryException {
+        try {
+            InputStream jsonNodesInputStream = ResourceLoader.getResourceFromLocations(jsonPath,
+                    this,
+                    "home:" + "/.config/cloud.io/",
+                    "file:/etc/cloud.io/",
+                    "classpath:cloud.io/");
+            byte[] bytes = jsonNodesInputStream.readAllBytes();
```

Comment:
AFAIK ObjectMapper can work on input streams too, this would be much more efficient than reading the whole file into an array.